robottelo/hosts.py

```diff
@@ -780,7 +781,9 @@ def contenthost_setup(
         repo_labels = repo_labels or []
         self.install_katello_ca(satellite)
         self.register_contenthost(org_label, activation_key=activation_key, lce=lce)
+        result = self.register_contenthost(org_label, activation_key=activation_key, lce=lce)
```

Comment:
I suppose this line should replace the one above...

Reply:
Yes!

Reply:
fixed, thank you